dmriprep/config/emc_coarse_Affine.json

```diff
@@ -0,0 +1,25 @@
+{
+    "dimension": 3,
+    "float": true,
+    "winsorize_lower_quantile": 0.002,
+    "winsorize_upper_quantile": 0.998,
+    "collapse_output_transforms": true,
+    "write_composite_transform": false,
+    "use_histogram_matching": [ false, false ],
+    "use_estimate_learning_rate_once": [ true, true ],
+    "transforms": [ "Rigid", "Affine" ],
+    "number_of_iterations": [ [ 100, 100 ], [ 100 ] ],
+    "output_warped_image": true,
+    "transform_parameters": [ [ 0.2 ], [ 0.15 ] ],
+    "convergence_threshold": [ 1e-06, 1e-06 ],
+    "convergence_window_size": [ 20, 20 ],
+    "metric": [ "Mattes", "Mattes" ],
```

Comment:
Is this choice based on speed?

Wouldn't it make more sense some cross-correlation based measure? MI is just going to exacerbate the problems you mentioned about the different content of each direction (pinging @arokem and @mattcieslak).

Reply:
I agree @oesteban -- cross-corr seems like it would be preferable here. Curious what @mattcieslak thinks?

Reply:
Remember, the fixed (model-estimated) images here will have the same contrast as the moving images so most metrics would be ok here. MI is much faster than CC, so any benefit of CC would be outweighed by its extra computation time. Also, this is only the coarse iteration.

Reply:
Yup, but MI has much flatter gradient - I'm going to make up a number here, but 1 CC iteration is as effective as 10 MI iterations.